pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -114,6 +114,7 @@ let
         done
         sed -i Makefile -e 's|= depmod|= ${buildPackages.kmod}/bin/depmod|'
         sed -i scripts/ld-version.sh -e "s|/usr/bin/awk|${buildPackages.gawk}/bin/awk|"
+        sed -i scripts/bpf_helpers_doc.py  -e "s|/usr/bin/env python3|${pkgs.buildPackages.python3}/bin/python3|"
```

Comment:
```suggestion
        patchShebangs scripts/ld-version.sh scripts/bpf_helpers_doc.py
```

Reply:
requires python3 to be in nativeBuildInputs.

Reply:
It's available under buildPackages, it seemed to work fine without it. 

Should I change the other two lines to use patchShebang too?

Reply:
Should `gawk` then be listed too?

Reply:
Yes. I would do the same for `gawk` too.